Cargo.toml

```diff
@@ -23,6 +24,8 @@ pretty_assertions = "1.2"
 regex = "1"
 serde = { version = "1.0", features = ["derive"] }
 serde-value = "0.7"
+tokio = { version = "1.20", default-features = false, features = ["macros", "rt-multi-thread"] }
```

Comment:
Is `rt-multi-thread` actually needed?  It's still fundamentally a sequential operation, unlike handling e.g. network requests.

Reply:
This is a test dependency and by default Rust runs all test in parallel, so I think it is needed. But I didn't dig deep, just copied from previous PRs

Reply:
> by default Rust runs all tests in parallel, so I think it is needed.

This is not relevant as far as I understand it as each test decorated with [`#[tokio::test]`](https://docs.rs/tokio/latest/tokio/attr.test.html) will run within its own runtime instance (which can be either multi- or single-threaded).

However, I think the multi-threaded flavour is the default IIRC and one would need to explicitly select the single-threaded flavour if only the `rt` feature is enabled, so I think that this might be simpler for a development dependency which does not burden downstream crates.

Reply:
Well, because both features are optional, I think we will use the least demanding feature, i.e. `"rt"`